packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -427,8 +430,8 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
    * Handle global tab presses so that we can patch tabindexes on the fly.
    */
   private _onKeyDownCapture = (ev: KeyboardEvent): void => {
-    if (ev.which === KeyCodes.tab) {
-      _outerZones.forEach(zone => zone._updateTabIndexes());
+    if (ev.key === 'Tab') {
```

Comment:
why this change to not use KeyCodes?

Reply:
`ev.which` and `ev.keyCode` are deprecated:

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
Ok would be good to remove the rest of this usage too but can be done in subsequent PRs as well

Reply:
What's the recommended alternative?

Reply:
Can we update the key codes enum or add a new one so we're not stuck using strings which aren't type safe?

Reply:
wait all. don't make a single code change in this pr. separate ones.

Reply:
Ok, I'll revert `which` and just add a tslint exception so that the linter doesn't complain. For reference, [these](https://github.com/OfficeDev/office-ui-fabric-react/pull/11765/commits/a34ca1ebef4c5f1248c5fa10e219fa8ce34b45d2) are the only things I changed.

Reply:
Here's what I was thinking:

Step 1. Single PR. Break out FZ into package. Don't remove old.
Step 2. Get it deployed and published. Make sure it's consumable and safe.
Step 3. Separate PR, now make OUFR depend on it.

I think we might be doing too many steps here.

Reply:
For sure, I'll remove what I did and just add a rule at the top to disable the linter in that file for now, since it won't pass the build if I don't disable it.

Reply:
BTW @khmakoto ev.key doesn't seem to be supported in android browser.

https://caniuse.com/#search=KeyboardEvent.key

Reply:
Also, we should adopt this eventually https://github.com/levithomason/keyboard-key/blob/master/src/keyboardKey.js

Reply:
Let's get the package to a v1 state, so that we can ship it in current without breaking everyone, then remove deprecations and stage that for a next release.

Reply:
Reverted the deprecation-related changes!